python/cuspatial/cuspatial/core/binpreds/feature_disjoint.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+from cuspatial.core.binpreds.binpred_interface import (
+    BinPred,
+    NotImplementedPredicate,
+)
+from cuspatial.core.binpreds.feature_intersects import (
+    IntersectsPredicateBase,
+    PointLineStringIntersects,
+)
+from cuspatial.utils.binpred_utils import (
+    LineString,
+    MultiPoint,
+    Point,
+    Polygon,
+)
+
+
+class ContainsDisjoint(BinPred):
+    def _preprocess(self, lhs, rhs):
+        """Disjoint is the opposite of contains, so just implement contains
+        and then negate the result."""
```

Comment:
That's a good idea. I probably have time for this.

Reply:
How is disjoint the opposite of contains for polygons? A polygon that is NOT disjoint from another polygon or a linestring does not necessarily contain it -- for example if they intersect.  Intersecting geometries are not disjoint and don't contain each other, except where the intersection is only a colinear point or segment.

Reply:
Thanks for catching this as well. Disjoint is the opposite of contains for (Point, Point), (Point, Polygon), (Polygon, Point), and (Polygon, MultiPoint). I mistakenly rushed into applying the same reversal to (Polygon, LineString) and (Polygon, LineString), possibly because I trusted my own incorrect docstring too much. I will modify the latter two types to `NotImplementedPredicate` and correct the docstring to better reflect the correct reversal.

As I write the remaining tests in the test_dispatch (next PR for 23.06) I will first make a list of the unimplemented cases (including the latter two here) and then cross-reference them all with the [Binary Predicate Design Doc](https://docs.google.com/document/d/19BzZ-C0e4vBl9tAMJeDUc9s6AhhAs8cYOGXkWCawbLY/edit?pli=1#heading=h.d8e4dm4w5mzd) rules. I'll use that for the rest of the more complicated relationships so I don't stumble down as many rabbit holes.

Reply:
[Enumerate non-simple types left to implement](https://github.com/rapidsai/cuspatial/issues/1037)